.github/workflows/deploy_production.yml

```diff
@@ -0,0 +1,225 @@
+          | Working Directory |  ${{ env.tf_actions_working_dir }} |
+          | Workflow          |  ${{ github.workflow }}            |
+          | Last updated      |  ${(new Date()).toUTCString()}     |
+          `;
+
+          // Call the script to write the comment
+          const script = require('./scripts/ghactions/comment-on-pr.js');
+          await script({github, context, core, commentTitle, commentBody});
+
+  terraform:
+    runs-on: ubuntu-latest
+    if: ${{ !startsWith(github.event.head_commit.message, 'skip-terraform:') }}
+    defaults:
+      run:
+        working-directory: ${{ env.tf_actions_working_dir }}
+    permissions:
+      contents: write
+      pull-requests: write
+    env:
+      TF_VAR_aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      TF_VAR_aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      TF_VAR_datadog_api_key: ${{ secrets.DATADOG_API_KEY}}
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 2
+
+    - uses: hashicorp/setup-terraform@v3
+      with:
+        terraform_wrapper: false
+
+    - name: Terraform fmt
+      id: fmt
+      run: terraform fmt -check
+      continue-on-error: true
+
+    - name: Terraform Init
+      id: init
+      run: terraform init
+
+    - name: Terraform Validate
+      id: validate
+      run: |
+        echo "terraform_validate<<EOF" >> "$GITHUB_OUTPUT"
+        echo "\$ terraform validate" >> "$GITHUB_OUTPUT"
+        terraform validate -no-color | tee -a "$GITHUB_OUTPUT"
+        echo "EOF" >> "$GITHUB_OUTPUT"
+
+    - name: Terraform Plan
+      id: plan
+      run: |
+        echo "terraform_plan<<EOF" >> "$GITHUB_OUTPUT"
+        echo "\$ terraform plan" >> "$GITHUB_OUTPUT"
+        terraform plan -no-color | tee -a "$GITHUB_OUTPUT"
+        echo "EOF" >> "$GITHUB_OUTPUT"
+      continue-on-error: true
+
+    - name: Terraform Apply
+      id: apply
+      run: |
+        echo "terraform_apply<<EOF" >> "$GITHUB_OUTPUT"
+        echo "\$ terraform apply -auto-approve" >> "$GITHUB_OUTPUT"
+        terraform apply -auto-approve -no-color | tee -a "$GITHUB_OUTPUT"
+        echo "EOF" >> "$GITHUB_OUTPUT"
+      continue-on-error: true
+
+    # TODO(art): this should only run on master merge
+    #- name: Commit and Push Terraform Outputs
+    #  id: commit
+    #  run: |
+    #    {
+    #      echo "git_commit<<EOF"
+    #      git config --global user.email "nothuman@ooni.org"
+    #      git config --global user.name "OONI Github Actions Bot"
+    #      git add .
+    #      git commit -m "skip-terraform: autoupdate terraform outputs" || echo "No changes to commit"
+    #      git push origin HEAD:${{ github.event.pull_request && github.head_ref || github.ref_name }}
+    #      echo "EOF"
+    #    } >> "$GITHUB_OUTPUT"
+    #  env:
+    #    GITHUB_REF: ${{ github.ref }}
+    #    GITHUB_HEAD_REF: ${{ github.head_ref }}
+    #  continue-on-error: true
```

Comment:
This has been moved to https://github.com/ooni/backend/pull/804. I should remove this comment and code

Reply:
This has been removed